non_linear/graphs/BFS.java

```diff
@@ -0,0 +1,68 @@
+
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.Queue;
+
+public class BFS {
+    public ArrayList<Integer> bfsOfGraph(int vertex,ArrayList<ArrayList<Integer>> adjacent){
+        ArrayList<Integer> bfs = new ArrayList<>();//the list stores the elements
+        boolean visit[] = new boolean[vertex];//visited array is created
```

Comment:
Comments can be better formatted as
`// Sentence.`

It's a good practice to leave a `space` after comment token. The upper and lower cases, can be handled the same way like how it's done in English. Comments can end with a fullstop.